how-to-guides/migration-from-5x.md

```diff
@@ -31,3 +32,13 @@ To use TinyMCE 6 from the {{site.cloudname}}, include this script in your HTML p
 ```
 
 For information on using the latest development and testing builds, see: [Cloud deployment guide - Specify editor & plugin versions]({{site.baseurl}}/cloud-deployment-guide/editor-plugin-version/).
+
+## Plugin Changes
+
+### Print plugin moved to TinyMCE Core
+
+The Print plugin has been moved to Core in {{site.productname}} 6.0.
+
+The print toolbar button, menu item, shortcut, and command will now work by default, without needing to specify `print` as a plugin.
```

Comment:
```suggestion
The `print` toolbar button, menu item, shortcut, and command will now work by default, without needing to specify `print` as a plugin.
```
Should this be in backticks since it refers to the name of the items?

Reply:
Hmm, I'm not sure on this one. @tylerkelly13 please advise. It's the same in the release notes too

Reply:
no backticks. the print shortcut isn't referred to as `print`, and `mcePrint` is the command. If they were all `print`, then I'd backtick.

Reply:
```suggestion
The toolbar button, menu item, shortcut, and command for printing content will now work by default, without needing to specify `print` as a plugin.
```

then, would this be better actually?

Reply:
Changed it to `The toolbar button, menu item, shortcut, and command for printing content are now available by default, without needing to specify `print` as a plugin.` here and in the release notes file. Let me know if I should change it, and if I should change the HR wording to match?

Reply:
Talked to Tyler in DMs, current wording is good